pkg/utils/database/mysql.go

```diff
@@ -195,3 +196,33 @@ func (m Mysql) GetCredentials() Credentials {
 		Password: m.Password,
 	}
 }
+
+// ParseAdminCredentials parse admin username and password of mysql database from secret data
+// If "user" key is not defined, take "root" as admin user by default
+func (m Mysql) ParseAdminCredentials(data map[string][]byte) (AdminCredentials, error) {
+	cred := AdminCredentials{}
+
+	_, ok := data["user"]
+	if ok {
+		cred.Username = string(data["user"])
+	} else {
+		// default admin username is "root"
+		cred.Username = "root"
+	}
+
+	// if "password" key is defined in data, take value as password
+	if ok {
```

Comment:
the check if password exist is missing, it's take the check from the username.

Reply:
oops thanks for catching it